### DISCUSSION TITLE
Import GPX navigation issue.

### DISCUSSION THREAD

**Discussion by SportTester** [osmandapp/OsmAnd#12358] [General]
Hi i create a GPX route that is 12km in length but when i load it and try to navigate the program does not follow the route that been displayed and goes to other roads that it should not and like to know why??????? When i plan routes in google map no problem. Its preventing me to follow the route in this app because of this issue.

**Comment by vshcherb** [comment]
Please give more details and share the track if possible

**Reply by SportTester** [reply]
Thanks I will do that!
